tests/spyre_util.py

```diff
@@ -527,11 +527,16 @@ def _get_or_default(env: str, default: str) -> str:
             "VLLM_SPYRE_TEST_MODEL_LIST",
             "sentence-transformers/all-roberta-large-v1")
         marks = [pytest.mark.embedding]
-    elif quantization == "gptq":
+    elif quantized == "gptq":
         # TODO: need a HF hub reference here as a default
         user_test_model_list = _get_or_default("VLLM_SPYRE_TEST_MODEL_LIST",
                                                "granite-3.0-8b-instruct-gptq")
         marks = [pytest.mark.decoder, pytest.mark.quantized, pytest.mark.spyre]
+    elif quantized == "fp8":
+        # TODO: need a HF hub reference here as a default
```

Comment:
Maybe I'm missing some context here but if I click on the `quantizations` link on the model page:

<img width="669" height="209" alt="Image" src="https://github.com/user-attachments/assets/0287134f-3177-4398-b19c-94c58c74811a" />

I see 2 models listed: https://huggingface.co/models?other=base_model:quantized:ibm-ai-platform/micro-g3.3-8b-instruct-1b

Reply:
I see that M Radermacher is quantizing a bunch of models? https://huggingface.co/mradermacher/micro-g3.3-8b-instruct-1b-GGUF

Seems like they work for https://www.nethype.de/

🤷